File: 3303/3303.c
// bee 3303 - Palavrao
// Alonso Martins
// 20/02/2024
#include <stdio.h>
#include <string.h>

int main() {
    char s[20];
    
    fgets(s, 20, stdin);
    
    printf(strlen(s) >= 10 ? "palavrao\n" : "palavrinha\n");
    
    return 0;
}

File: 3303/3303.java
// bee 3303 - Palavrao
// Alonso Martins
// 20/02/2024
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String s = in.nextLine();
        System.out.println(s.length() >= 10 ? "palavrao" : "palavrinha");
    }
}
